core/model/src/main/java/org/activityinfo/model/query/ColumnView.java

```diff
@@ -0,0 +1,30 @@
+package org.activityinfo.model.query;
+
+import java.util.Date;
+
+public interface ColumnView {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move constants to a class or enum. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1214)